migrate/20250723_add_aws_amis_with_cloudwatch.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  up do
+    run <<~SQL
+      UPDATE pg_aws_ami SET aws_ami_id = 'ami-0d95b37f7cd853a2d' WHERE aws_location_name = 'us-west-2' AND pg_version = '16' AND arch = 'x64';
```

Comment:
Can you also add images for ap-southeast-2. We are planning to enable that region soon.

Reply:
Added, please check